doc/sphinx/cython/units.rst

```diff
@@ -11,6 +11,7 @@ Solution with Units
 -------------------
 
 .. autoclass:: Solution
+   :no-members:
```

Comment:
I believe this is done in the class docstrings?

Reply:
Not necessarily. A link to the regular `Solution` class would be necessary to access the documentation of methods etc.?

Reply:
I don't disagree with you... And I want so badly for this PR to be done 😅 can you add what you're thinking of, rather than going back and forth through another round of reviews?